src/Platform/Microsoft.Testing.Platform/Messages/ChannelConsumerDataProcessor.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Microsoft.Testing.Platform.Messages;
 
-[DebuggerDisplay("DataConsumer = {DataConsumer.Uid}")]
+[DebuggerDisplay("DataConsumer = {DataConsumer.Value}")]
```

Comment:
this is `uid` I think https://github.com/microsoft/testfx/blob/890efd91bd8d6863d12de28892344edf33c31543/src/Platform/Microsoft.Testing.Platform/Extensions/IExtension.cs#L15

Reply:
Thanks! Stupid VS rename...

```suggestion
[DebuggerDisplay("DataConsumer = {DataConsumer.Uid}")]
```